src/intents/Declaration.ts

```diff
@@ -101,6 +102,18 @@ class Declaration {
     return bits
   }
 
+  nonces (): BitNoncePair[] {
```

Comment:
the method is named `nonces` but returned both `bits` and `nonces` i wonder if it's good naming? 🤔

Reply:
Maybe `bitAndNonces`? `bitNoncePair`? 🤔  Another recommendation?